packages/erxes-ui/src/components/richTextEditor/RichTextEditorControl/TableControl.tsx

```diff
@@ -9,24 +8,20 @@ import {
   RichTextEditorControlBase,
 } from './RichTextEditorControl';
 import Icon from '../../Icon';
-import TableMenuControls from './TableMenuControls';
 
-type BubbleMenuShowProps = {
+export type BubbleMenuShowProps = {
   editor: Editor;
   view: EditorView;
   state: EditorState;
   oldState?: EditorState;
   from: number;
   to: number;
 };
-
 const TableIcon: IRichTextEditorControlBaseProps['icon'] = () => (
   <Icon icon="table" />
 );
-
 export function TableControl() {
   const { editor, labels } = useRichTextEditorContext();
-
   const handleTableRequest = () => {
     editor
       ?.chain()
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Review the necessity of optional chaining in editor commands.

The use of optional chaining (`?.`) with `editor.chain()` suggests that `editor` might be null or undefined. If the editor instance is always expected to be present when this function is called, consider asserting its presence instead of using optional chaining.

```suggestion
.chain()
```